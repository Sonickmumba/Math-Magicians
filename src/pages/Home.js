import React from 'react';

const Home = () => (
  <main id="home-link-container">
    <h2 className="home-title">Welcome To Our Page</h2>
    <p className="home-link-text">
      Welcome to the Math Magician application! Get ready to unlock the secrets 
      of numbers, unravel complex equations, and explore the fascinating world of 
      mathematics. Whether you're looking for quick calculations, mathematical insights, 
      or challenging problems, I'm here to assist you on your magical math journey. Let's 
      embark on this adventure together and make math fun and enchanting!
    </p>
    <p className="home-link-text">
      
    </p>
  </main>
);

export default Home;
