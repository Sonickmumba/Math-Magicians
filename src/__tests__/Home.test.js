import { render, screen } from '@testing-library/react';
import TestRenderer from 'react-test-renderer';
import Home from '../pages/Home';

test('Test for Home Component', () => {
  render(<Home />);
  const linkElement = screen.getByText(/Welcome To Our Page/i);
  expect(linkElement).toBeInTheDocument();
});

describe('Test for  Home component', () => {
  it('Home Component renders correctly', () => {
    const tree = TestRenderer.create(<Home />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
