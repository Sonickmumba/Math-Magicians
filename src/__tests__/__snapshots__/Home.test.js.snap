// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test for  Home component Home Component renders correctly 1`] = `
<main
  id="home-link-container"
>
  <h2
    className="home-title"
  >
    Welcome To Our Page
  </h2>
  <p
    className="home-link-text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
  <p
    className="home-link-text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</main>
`;

exports[`Unit Test For  Home component Home Component renders correctly 1`] = `
<main
  id="home-link-container"
>
  <h2
    className="home-title"
  >
    Welcome To Our Page
  </h2>
  <p
    className="home-link-text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
  <p
    className="home-link-text"
  >
    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
  </p>
</main>
`;
